### DISCUSSION TITLE
Any way to set default quality for next/image?

### DISCUSSION THREAD

**Discussion by Mistwell** [vercel/next.js#19346] [Help] [answered]
Need to set defult quality 100  for all imgs because of corrupted some img with 75.

**Comment by erwanvivien** [comment, marked as answer]
I hope you found an answer by now but :

```js
<Image
  alt={item.alt}
  src={item.img}
  quality={100}
/>
```

Works :)

**Reply by leerob** [reply]
Going further, you can just wrap the `next/image` component with your own component, with the quality prop hardcoded to 100 if you want 😄

**Reply by miguderp** [reply]
Way easier than my idea below, duh!

**Comment by miguderp** [comment]
I haven’t tested it myself but potentially if you don’t want to have to add `quality={100}` on every `<Image />` tag, you could override the loader in your `next.config.js` to specify a custom loader with the quality as a query parameter (set to 100); [read more](https://nextjs.org/docs/api-reference/next/image#loader).

Instead of doing it on an image basis, you can specify it in your `next.config.js` file; [read more](https://nextjs.org/docs/basic-features/image-optimization#loader)

You might need to adapt the URL so it includes `/_next/image?url=` for the image optimization is still taken care of, something like https://domain.tld/_next/image?url=
